File: Programmers/Lv.0/원소들의 곱과 합/181929.js
function solution(num_list) {
    let answer = 0;
    
    let m = 1, s = 0;
    for (const num of num_list) {
        m *= num;
        s += num;
    }
    
    answer = (m < s**2) ? 1 : 0;
    
    return answer;
}

File: Programmers/Lv.0/원소들의 곱과 합/181929.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        int m = 1, s = 0;
        for (int num : num_list) {
            m *= num;
            s += num;
        }
        
        answer = (m < s * s) ? 1 : 0;
        
        return answer;
    }
}
